src/github/api/read.rs

```diff
@@ -112,7 +115,7 @@ impl GithubRead for GitHubApiRead {
         let mut owners = HashSet::new();
         self.client.rest_paginated(
             &Method::GET,
-            format!("orgs/{org}/members?role=admin"),
+            &GitHubUrl::orgs(org, "members?role=admin")?,
```

Comment:
I decided to introduce a new struct (GitHubUrl) instead of passing the org as a separate parameter.
It also avoids repeating the start of the endpoint (`orgs/{org}`) every time.

The con is that it adds a bit of complexity.

I hope you like it.